explainer.md

```diff
@@ -689,6 +701,7 @@ dictionary XRWebGLLayerInit {
   boolean depth = true;
```

Comment:
Per our conversation earlier about naming confusion... Perhaps these become "depthBuffer" or "alphaEnabled"?

Reply:
I'm highly reluctant to do so because currently these match, in both verbiage and function, the arguments from WebGL context creation exactly.

Reply:
Yeah.  That makes sense...